core/src/main/java/tc/oc/pgm/PGMPlugin.java

```diff
@@ -273,19 +277,33 @@ public void reloadConfig() {
     final Logger logger = getLogger();
     logger.setLevel(config.getLogLevel());
 
-    for (String source : config.getMapSources()) {
-      final MapSourceFactory factory;
+    for (String source : config.getFolderMapSources()) {
       try {
-        factory =
-            source.equalsIgnoreCase("default")
-                ? DefaultMapSourceFactory.INSTANCE
-                : new SystemMapSourceFactory(source);
+        if (new File(source).exists()) {
+          mapSourceFactories.add(new SystemMapSourceFactory(source));
+        }
       } catch (Throwable t) {
         t.printStackTrace();
-        continue;
+        logger.log(Level.WARNING, "Map source " + source + " is invalid and will be ignored");
       }
+    }
 
-      mapSourceFactories.add(factory);
+    try { // Disable if JGit is not present
+      if (Class.forName("org.eclipse.jgit.api.Git") != null) {
+
+        for (GitRepository source : config.getGitMapSources()) {
+          try {
+            mapSourceFactories.add(new GitMapSourceFactory(source, logger));
+          } catch (Throwable e) {
+            e.printStackTrace();
+            logger.log(
+                Level.WARNING,
+                "Map source " + source.getDir().getName() + " is invalid and will be ignored");
+          }
+        }
+      }
+    } catch (ClassNotFoundException e) {
+      logger.log(Level.SEVERE, "Unable to load JGit(was it excluded when compiling the jar?)");
```

Comment:
Don't log an error here, just `// No-op`

Reply:
Maybe not severe, but leaving in a warning that you're trying to load a git repo on a PGM build that does not include JGit seems pretty important, otherwise it can be hard to figure out why your repo isn't loading

Reply:
`Git is not loaded (are you running a custom version of PGM without JGit?)`